documentation/SdkModeExplained.md

```diff
@@ -1,32 +1,27 @@
+[Back to main page](../README.md)
+
 # SDK Mode
 
-The SDK mode is the mode of the sensor in which the wider range of stream capabilities are offered, i.e higher sampling rates, wider (or narrow) ranges etc. For example, in the SDK mode the accelerometer sampling rate can be chosen from values 26Hz, 52Hz, 104Hz, 208Hz or 416Hz compared to 52Hz available in normal operation mode. 
+The SDK mode is the mode of the device in which a wider range of stream capabilities are offered, i.e higher sampling rates, wider (or narrow) ranges etc. 
+
+> [!WARNING]
+> When SDK mode is enabled, all algorithms inside the device are disabled to enable free choice of settings for the sensors. It means that any computed data such as heart rate, PP intervals, RR intervals or any passively gathered data such as activity or sleep data will not be gathered anymore.
```

Comment:
is this true for both sensors and wrist units devices?

Reply:
by sensors I meant more the 9D sensors (acc, gyro, mag) and ECG / PPG. As this seems to be confusing, what other word could be used instead of that ?

Reply:
Is this any clearer:
 all algorithms on the device are disabled as they work with only certain device sensor settings.

Reply:
that does sound clearer. I'll change it this way. Thanks for the suggestion.